pkg/log/log_messages.go

```diff
@@ -80,3 +80,12 @@ const CAKC064 string = "CAKC064 Unable to find Authenticator for this configurat
 const CAKC065 string = "CAKC065 File path is invalid %s"
 const CAKC066 string = "CAKC066 Performing authn-jwt"
 const CAKC067 string = "CAKC067 Failed reading jwt token from %s"
+const CAKC068 string = "CAKC068 Loading config object..."
```

Comment:
Hey @szamir1 @shulifink 
May you go over the logs here please?

Reply:
Maybe "Loading configuration object..." to be consistent with 069 and 070